arb/dump_file.c

```diff
@@ -0,0 +1,31 @@
+/*
+    Copyright (C) 2019 Julian Rüth
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdio.h>
+#include <string.h>
+
+#include "arb.h"
+
+int arb_dump_file(FILE* stream, const arb_t x)
```

Comment:
This is essentially copy & pasted twice more (for arf & mag). Where should I put a `str_dump_file(FILE* stream, const char* data)` so it can be used from all three functions? Or should I keep this duplication?